simulator/virtual_machine_test.go

```diff
@@ -2107,3 +2108,79 @@ func TestVmRefreshStorageInfo(t *testing.T) {
 		t.Errorf("expected %d, got %d", fileLayoutExCount, len(vmm.LayoutEx.File))
 	}
 }
+
+func TestApplyExtraConfig(t *testing.T) {
+	ctx := context.Background()
+
+	m := VPX()
```

Comment:
Note that you can use `Test(func(ctx context.Context, c *vim25.Client) {})` to take care of the setup+teardown. For example: https://github.com/vmware/govmomi/blob/3e64efd945ac46096ede2db22ebb987f5131ed29/simulator/virtual_machine_test.go#L1619-L1621

Reply:
I remembered that, but ended up copying a pattern from the same file. I'll use your recommendation.